docs/antora.yml

```diff
@@ -13,9 +13,9 @@ asciidoc:
     compatible-rhel7: '7.x'
     compatible-rhel8: '8.x'
     compatible-ubuntu: '20.04 LTS'
-    compatible-debian: '9 - 10'
+    compatible-debian: '9 - 11'
     compatible-opennms-helm: '3+'
```

Comment:
I'm not sure if we want people to indirectly encourage them to use this old version :)  Maybe we say `7.x` instead of `3+` as the minimal version.

Reply:
Yeah, it pretty much needs to be 7 now anyway.  I'd make it 7+